src/app/cli/src/init/client.ml

```diff
@@ -2297,7 +2293,7 @@ let signature_kind =
              (* Prefix string to disambiguate *)
              "other network: " ^ s
        in
-       Core.print_endline signature_kind_string )
+       print_endline signature_kind_string )
```

Comment:
The reason why CI broke is that this is a `Command.basic` and not a `Command.async`, so I'm pretty sure the async scheduler is not active here. The existing `Core.print_endline` just prints to stdout, so it works. This `print_endline` is the async one and tries to add the write to the async queue, so nothing ends up being printed - I guess just silently does nothing when the async scheduler isn't active?

Reply:
I'll try to rebase this atop compatible: now it's taking forever to load nix on this branch

Reply:
Also, I'm building with `devnet` profile. Although `devnet` seems to be the samething as `testnet`, it's confusing

Reply:
I can build with this though
![image](https://github.com/user-attachments/assets/b8ea782d-e9a9-4f50-994b-a7c03b03406f)